Source/population_internal.h

```diff
@@ -28,14 +28,14 @@ class population_internal : population {
     ~population_internal() override;
 
     // Returns the number of creates in the population.
-    size_t get_size() override;
+    [[nodiscard]] size_t get_size() const override;
```

Comment:
What is `[[nodiscard]]`?

Reply:
nodiscard means you should not delete the return type